### PR TITLE
[ja] Localize Nodes for Japanese

### LINKED ISSUE #2719: [ja] Localize Nodes for Japanese
Localize https://glossary.cncf.io/nodes/

### PR DESCRIPTION
**Pull request by Okabe-Junya**
### Describe your changes

Localize Nodes for Japanese

### Related issue number or link (ex: `resolves #issue-number`)

resolves #2719 

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/ja/nodes.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+アプリケーションは1台のマシン上で動作させることができます(そして実際に多くのアプリケーションがそうしています)が、それにはいくつかのリスクが伴います。
+具体的には、基盤となるシステムの故障がアプリケーションの中断を引き起こすことです。
+これに対処するために、開発者たちは[分散アプリケーション](/ja/distributed-apps/)を作り始めました。
+これは、各プロセスがそれぞれのノード上で動作します。
+それゆえ、ノードはアプリやプロセスを実行し、共通の目標を達成するために協力するノードの[クラスター](/ja/cluster/)、またはグループの一部として機能します。
+
+## どのように役に立つのでしょうか
+
+ノードはクラスタに割り当てることができる計算上の明確な単位(メモリ、CPU、ネットワーク)を提供します。
+[クラウドネイティブ](/cloud-native-tech/)プラットフォームやアプリでは、ノードは作業を行うことができる単一のユニットを表します。
```
[the remark this answers is not in the document]
Comment (Okabe-Junya) [reply content/ja/nodes.md]: すいません、ご指摘ありがとうございます。

https://github.com/cncf/glossary/pull/2725/commits/a624cf0a1d1c2742ea2755b967e24627b01e2c89 で修正しました

File: content/ja/nodes.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+アプリケーションは1台のマシン上で動作させることができます(そして実際に多くのアプリケーションがそうしています)が、それにはいくつかのリスクが伴います。
+具体的には、基盤となるシステムの故障がアプリケーションの中断を引き起こすことです。
+これに対処するために、開発者たちは[分散アプリケーション](/ja/distributed-apps/)を作り始めました。
+これは、各プロセスがそれぞれのノード上で動作します。
+それゆえ、ノードはアプリやプロセスを実行し、共通の目標を達成するために協力するノードの[クラスター](/ja/cluster/)、またはグループの一部として機能します。
+
+## どのように役に立つのでしょうか
+
+ノードはクラスタに割り当てることができる計算上の明確な単位(メモリ、CPU、ネットワーク)を提供します。
```
[the remark this answers is not in the document]
Comment (Okabe-Junya) [reply content/ja/nodes.md]: ありがとうございます、「分離」という意味を確かに暗黙的に表現している形になっていましたね。。

https://github.com/cncf/glossary/pull/2725/commits/849a55b6ccd7fcc75c8301e6ef4adad969b33a8e で修正しました。

**Comment by Okabe-Junya** [comment]
@naonishijima 

いつもレビューありがとうございます。確認、修正を行いましたので、再度ご確認いただけますと幸いです

**Review by naonishijima — APPROVED** [review]
LGTM

**Review by kaitoii11 — APPROVED** [review]
LGTM

**Pull request merged**